src/ripple/core/ConfigSections.h

```diff
@@ -55,7 +55,6 @@ struct ConfigSection
 #define SECTION_PATH_SEARCH_MAX         "path_search_max"
 #define SECTION_PEER_PRIVATE            "peer_private"
 #define SECTION_PEERS_MAX               "peers_max"
-#define SECTION_RPC_ADMIN_ALLOW         "rpc_admin_allow"
```

Comment:
Should we warn if the config file contains an obsolete `rpc_admin_allow` section? I'm thinking that would be a different PR (warn if the config file contains unknown sections).